lib/puppet/provider/service/upstart.rb

```diff
@@ -1,5 +1,3 @@
-require 'semver'
```

Comment:
We don't have to be that conservative when we have had a deprecation for a Ruby API. Still nice to reach out to module authors if we can find the modules that need an updated.